src/wallet/rpcwallet.cpp

```diff
@@ -3213,7 +3213,7 @@ UniValue spendallzerocoin(const UniValue& params, bool fHelp) {
     return  hasUnspendableMints;
 }
 
-UniValue spendzerocoinV3(const UniValue& params, bool fHelp) {
+UniValue spendSigma(const UniValue& params, bool fHelp) {
```

Comment:
In keeping with the standard for RPC functions, they should all be lower case.

Reply:
done